packages/fast-development-site-react/src/components/breadcrumb/breadcrumb-item.tsx

```diff
@@ -1,23 +1,24 @@
 import * as React from "react";
 import { Link } from "react-router-dom";
 import { IDevSiteDesignSystem } from "../design-system";
+import { toPx } from "@microsoft/fast-jss-utilities";
 import manageJss, { ComponentStyles, IJSSManagerProps, IManagedClasses } from "@microsoft/fast-jss-manager-react";
 
 export interface IBreadcrumbItemProps {
     to: string;
 }
 
 export interface IBreadcrumbItemManagedClasses {
-    breadcrumb_listItem: string;
+    breadcrumbItem_listItem: string;
 }
 
 const style: ComponentStyles<IBreadcrumbItemManagedClasses, IDevSiteDesignSystem> = {
-    breadcrumb_listItem: {
-        "paddingRight": "12px",
+    breadcrumbItem_listItem: {
```

Comment:
@nicholasrice This is an interesting one. When used within the markup, this seems correct. But this is a component itself. I would not expect us to follow a nested class convention here. It seems confusing.

Reply:
how is it a component itself?

A block only ever creates classes for its own block. It will never create classes outside of its block - if there are classes specific to the list-item block then then list-item needs to create those classes.

Reply:
cool - I've reset my review.